Sourcery/Parsing/FileParser.swift

```diff
@@ -702,7 +702,7 @@ extension FileParser {
                     var attributeRange = prefix.range(of: $0.description, options: .backwards)
                     // we expect all attributes to be prefixed with `@`
```

Comment:
Maybe it's time to add property like `requiresAtPrefix` (can't find a better name ^^) in the `Attribute` type and hide this check there? We can also reuse it in `name` property of `Attribute` type.